autoprof/fit/lm.py

```diff
@@ -370,38 +371,38 @@ def update_J_AD(self):
         self.full_jac = True
             
     @torch.no_grad()
-    def update_J_Broyden(self, h, Yp, Yph):
+    def update_J_Broyden(self, h, Yp, Yph)-> None:
         self.J = Broyden_step(self.J, h, Yp, Yph)
         if self.model.target.has_mask:
             self.J[self.mask] = 0.
         self.full_jac = False
 
     @torch.no_grad()
-    def update_hess(self):
+    def update_hess(self)-> None:
         if isinstance(self.W, float):
             self.hess = torch.matmul(self.J.T, self.J)
         else:
             self.hess = torch.matmul(self.J.T, self.W.view(len(self.W),-1)*self.J)
         self.hess += self.epsilon5 * torch.eye(len(self.current_state), dtype = AP_config.ap_dtype, device = AP_config.ap_device)
             
     @torch.no_grad()
-    def covariance_matrix(self):
+    def covariance_matrix(self)-> torch.Tensor:
         try:
             return torch.linalg.inv(self.hess)
         except:
-            AP_config.ap_logger.warning("WARNING: Hessian is singular, likely at least one model is non-physical. Will massage Hessian to continue but results should be inspected.")
+            AP_config.ap_logger.warning("WARNING: Hessian is singular, likely at least one model is non-physical. Will message Hessian to continue but results should be inspected.")
             self.hess += torch.eye(len(self.grad), dtype = AP_config.ap_dtype, device = AP_config.ap_device)*(torch.diag(self.hess) == 0)
             return torch.linalg.inv(self.hess)
             
     @torch.no_grad()
-    def update_grad(self, Yph):
+    def update_grad(self, Yph)-> None:
         self.grad = torch.matmul(self.J.T, self.W * (self.Y - Yph))
             
     @torch.no_grad()
-    def rho(self, Xp, Xph, h):
+    def rho(self, Xp, Xph, h)-> torch.Tensor:
         return self.ndf*(Xp - Xph) / abs(torch.dot(h, self.L * (torch.abs(torch.diag(self.hess) - self.epsilon5) * h) + self.grad))
 
-    def accept_history(self):
+    def accept_history(self)-> (torch.Tensor, torch.Tensor, List[float]):
```

Comment:
All three will be lists, so the return type is `(List[np.ndarray], List[np.ndarray], List[float])`